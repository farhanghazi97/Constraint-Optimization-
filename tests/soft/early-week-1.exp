m1:thu 12pm
m2:fri 9am
m3:mon 12pm
cost:7
