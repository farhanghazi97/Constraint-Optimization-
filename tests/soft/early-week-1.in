# early week soft constraint
meeting, m1
meeting, m2
meeting, m3
domain, m1, early-week, soft
domain, m1, thu 12pm-fri 12pm, hard
domain, m2, early-week, soft
domain, m2, after thu, hard
domain, m3, early-week, soft
domain, m3, afternoon, hard
