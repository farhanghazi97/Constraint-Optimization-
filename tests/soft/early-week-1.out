m1:thu 1pm
m2:fri 9am
m3:mon 2pm
cost:7
